catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -249,13 +249,13 @@ public void init() {
         try {
             initCorba();
         } catch (InvalidName | AdapterInactive | WrongPolicy | ServantNotActive e) {
-            LOGGER.warn("Unable to start the CORBA server. {}",
+            LOGGER.info("Unable to start the CORBA server. Set to DEBUG for a full stack trace {}",
```

Comment:
I think it is fine if you have something that you think is important to see, but might happen often enough that it could spam the log with stack traces.